src/strategies/spidStrategy.ts

```diff
@@ -11,6 +11,24 @@ import { SpidUser } from "../types/user";
 import { fetchIdpMetadata, parseIdpMetadata } from "../utils/idpLoader";
 import { log } from "../utils/logger";
 
+export const remapIpdMetadata = (
+  // tslint:disable-next-line: no-any
+  options: { idp: { [key: string]: IDPEntityDescriptor }; sp: any },
+  idpMetadata: ReadonlyArray<IDPEntityDescriptor>,
+  idpIds: { [key: string]: string | undefined }
+) => {
+  idpMetadata.forEach(idp => {
+    const IDP_KEY = idpIds[idp.entityID];
```

Comment:
all caps identifiers are used only for constant values, use camel case

Reply:
fixed in 7d7e64b